README.md

```diff
@@ -214,10 +214,15 @@ The PIConGPU Team expresses its thanks to:
 - Stefan Tietze
 - Benjamin Worpitz
 - Erik Zenker
+- Alexander Matthes
 
 Kudos to everyone who helped!
 
 ********************************************************************************
 
 ![image of an lwfa](doc/images/lwfa_grey.png "LWFA")
```

Comment:
As I added two clipping planes here, you are in fact very right. But sometimes a 2D visualization just makes sense to understand things. ;)
In the end: Your decision.